app/javascript/components/analytics/enroller/EnrollerAnalytics.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Row, Col } from 'react-bootstrap';
+import EnrollerStatistics from './EnrollerStatistics';
+
+class EnrollerAnalytics extends React.Component {
+  constructor(props) {
```

Comment:
you removed this somewhere else, so is this necessary?

Reply:
good call: https://github.com/SaraAlert/SaraAlert/pull/993/commits/997e55e1b9d92783360251219d6249035d610ed6